plugin/al/usdmayautils/AL/usdmaya/utils/MeshUtils.cpp

```diff
@@ -763,17 +934,16 @@ void MeshImportContext::applyPrimVars(bool createUvs, bool createColours)
     SdfValueTypeName typeName;
     int elementSize;
     primvar.GetDeclarationInfo(&name, &typeName, &interpolation, &elementSize);
```

Comment:
Not really sure why we creating `typeName `and `elementSize ` objects that are not used?
Can't just pass nullptr for the arguments we don't need to be populated?
`primvar.GetDeclarationInfo(&name, nullptr , &interpolation, nullptr );`

Reply:
Just ran tests, and I get a runtime error "Coding Error: in GetDeclarationInfo at line 188 of /USD-19.11/pxr/usd/lib/usdGeom/primvar.cpp -- Failed verification: ' name && typeName && interpolation && elementSize '" ... so have reverted that change